packages/next/next-server/lib/router/router.ts

```diff
@@ -676,16 +678,18 @@ export default class Router implements BaseRouter {
     })
   }
 
-  _getStaticData = (asPath: string, _cachedData?: object): Promise<object> => {
-    let pathname = parse(asPath).pathname
+  _getStaticData = (asPath: string, cache?: boolean): Promise<object> => {
```

Comment:
It might make sense to have a new `_getServerData` method instead of sharing this one.

Reply:
Added a `_getServerData` method to separate them out